### DISCUSSION TITLE
REPETICÃO DE VIDEOS

### DISCUSSION THREAD

**Discussion by flavio085** [aclap-dev/video-downloadhelper#834] [Help and Q&A]
### Your question

COMO FAÇO PARA NAO APARECER SEMPRE OS MESMOS VIDEOS?????????????????????????

### Addon & CoApp version and details

COMO FAÇO PARA NAO APARECER SEMPRE OS MESMOS VIDEOS?????????????????????????

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I am just another user.

You need to actually go through the steps required when you created this discussion.  The process guides you through the steps you are supposed to follow.  Go back & do it again, but this time, don't skip the instructions.  Read the instructions & follow them.
